docs/_toc.yml

```diff
@@ -32,6 +32,8 @@ parts:
       file: model_embeddings
     - title: Finetuning
       file: model_finetuning
+    - title: Partial inputs
+      file: partial-inputs
 - caption: Tutorials
```

Comment:
Should this go under the tutorials section below?

Reply:
absolutely